package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "bootstrap": "^5.2.0"
```

Comment:
If I'm not missing something you are not using bootstrap from npm.
The `index.html` uses links to CDN, so this dependency could be removed.